packages/@aws-cdk/aws-rds/lib/cluster-engine.ts

```diff
@@ -49,6 +49,36 @@ export interface ClusterEngineConfig {
    * @default - use the default port for clusters (3306)
    */
   readonly port?: number;
+
+  /**
+   * Features supported by the database engine.
+   *
+   * @see https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/API_DBEngineVersion.html
+   *
+   * @default - no features
+   */
+  readonly features?: EngineFeatures;
+}
+
+/**
+ * Represents Database Engine features
+ */
+export interface EngineFeatures {
```

Comment:
`ClusterEngineFeatures`? We might have `InstanceEngineFeatures` in the future as well.

Reply:
i like it.makes sense with our current usage

One reason I was keeping it general was because the supported features list was essentially retrieved from running `aws rds describe-db-engine-versions` and the documentation wasn't especially specific on where they're used.